src/app/client/src/app/app.component.ts

```diff
@@ -125,7 +125,10 @@ export class AppComponent implements OnInit, OnDestroy {
   isStepperCompleted = false;
   OnboardingFormConfig: any;
   isStepperEnabled = false;
-  isPopupEnabled = false;
+  onboardingData: any;
+  isOnboardingEnabled = true;
+  isBmgEnabled = true;
```

Comment:
@mithun30052001 rename  isBmgEnabled to isFWSelectionEnabled

Reply:
sure @princegupta1131 i will change